ingestion/src/metadata/utils/helpers.py

```diff
@@ -44,6 +49,39 @@
 logger = logging.getLogger(__name__)
 
 
+@singledispatch
+def get_connection_url(serviceConnectionConfig):
+    raise ValueError(f"Hola {serviceConnectionConfig}")
+
+
+@get_connection_url.register(MysqlConnection)
+def _(serviceConnectionConfig):
```

Comment:
let's clean this up a bit. We could do something like:

```
@get_connection_url.register
def _(connectionConfig: MysqlConnection):
    ...
```
and just pick the attributes without casting to dict, not needed with proper typing of the arguments